build.js

```diff
@@ -0,0 +1,128 @@
+var upload = require('prebuild/upload')
+var spawn = require('child_process').spawn
+var async = require('async')
+var extend = require('xtend')
+var fs = require('fs')
+var path = require('path')
+var mkdirp = require('mkdirp')
+var tar = require('tar-stream')
+var zlib = require('zlib')
+var pkg = require('./package.json')
+
+function mode(octal) {
+  return parseInt(octal, 8)
+}
+
+var arch = process.env.ARCH
+  ? process.env.ARCH
+    .replace('i686', 'ia32')
+    .replace('x86_64', 'x64')
+  : process.arch
+
+var cmakeJsPath = path.join(
+  __dirname,
+  'node_modules',
+  '.bin',
+  process.platform === 'darwin' ? 'cmake-js' : 'cmake-js.cmd'
+)
+
+var support = require('./support')
+var targets = support.targets
+var abis = support.abis
+var files = []
+
+var chain = Promise.resolve()
+targets.forEach(parts => {
```

Comment:
Loops over the targets in the "support.js" files, and builds all of those one after the other, when they are all done, it uploads the files to github, on AppVeyor, make sure to setup the GITHUB_ACCESS_TOKEN (I think) key that has access to your repo so it can upload the builds